employee.html

```diff
@@ -0,0 +1,215 @@
+{% extends 'base.html' %}
+
+{% load humanize %}
+{% load staticfiles %}
+
+{% block toolbar %}
+    {% include 'searchbar.html' %}
+{% endblock %}
+{% block content %}
+    <div class="employee padded">
+        <div class="row">
+            <div class="col-4">
+                <div class="basic-info">
+                    <h1 class="name">{{ employee.first_name }} {{ employee.last_name }}</h1>
+                    <h2 class="position">{{ employee.position }}</h2>
+                    <h3 class="dept">{{ employee.full_department }}</h3>
+                </div>
+
+                <div class="tabs">
+                    <ul class="tab-links">
+                        <li class="tab1"><a href="javascript:void(0)" class="tab-links" onclick="openSalary(event, 'tab1')"
+                          id="defaultOpen">2016</a>
+                        <li class="tab2"><a href="javascript:void(0)" class="tab-links" onclick="openSalary(event, 'tab2')">2015</a>
+                        <li class="tab3"><a href="javascript:void(0)" class="tab-links" onclick="openSalary(event, 'tab3')">2014</a>
+                    </ul>
+                </div>
+
+                <!-- TAB STYLE -->
+                <style>
+                h3.dept {
```

Comment:
Ideally this css should go in the static css file (salarydb/static/css/styles.css).